### PR TITLE
My First Pull Request

### PR DESCRIPTION
**Pull request by lusekayz**
This is my first Microverse Project built using HTML and CSS

### CODE REVIEW

File: .github/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (arslanbisharat) [inline .github/linters.yml:1]: - [ ] You have added linters in your project but on the Github actions it not working. Please make sure they are working on your pull request.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @lusekayz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/linters.yml (hunk 1 above)
Comment (lusekayz) [reply .github/linters.yml:1]: Hie @arslanbisharat I ran the linters and got "This branch has no conflict with the base branch". I don;t understand where I went wrong according to your review. Please assist.

**Comment by lusekayz** [comment]
@arslanbisharat I have made the requested changes. Please review once more. Thanks

**Comment by lusekayz** [comment]
@arslanbisharat I have made the requested changes. Please review once more. Thanks

**Comment by arslanbisharat** [comment]
> @arslanbisharat I have made the requested changes. Please review once more. Thanks

Hi, 
A new CR will look into your project. But I can see still linters aren’t working

File: .github/linters.yml (hunk 1 above)
Comment (arslanbisharat) [reply .github/linters.yml:1]: It is okay to have conflict between PR

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+#Getting Started
+
+This is my First Microverse Project
+
+To get a local copy up and running follow these simple example steps.
+Prerequisites
+Setup
+Install
+Usage
+Run tests
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] Your README file is well designed but you are not fully compliant with the README template provided from the Microverse. Please review this [README file](https://github.com/microverseinc/readme-template/blob/master/README.md) and make any necessary changes!

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @lusekayz , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ GitHub flow was set up correctly
✔️ Good pull request title and description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @lusekayz , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**